subprojects/tooling-api/src/integTest/groovy/org/gradle/integtests/tooling/ToolingApiDeprecatedClientJvmCrossVersionSpec.groovy

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package org.gradle.integtests.tooling
+
+import org.apache.commons.lang3.StringUtils
+import org.gradle.integtests.fixtures.AvailableJavaHomes
+import org.gradle.integtests.fixtures.ScriptExecuter
+import org.gradle.integtests.tooling.fixture.TargetGradleVersion
+import org.gradle.integtests.tooling.fixture.ToolingApiSpecification
+import org.gradle.integtests.tooling.fixture.ToolingApiVersion
+import org.gradle.util.GradleVersion
+import spock.lang.IgnoreIf
+
+@IgnoreIf({ AvailableJavaHomes.jdk7 == null })
+class ToolingApiDeprecatedClientJvmCrossVersionSpec extends ToolingApiSpecification {
+    def setup() {
+        settingsFile << "rootProject.name = 'test'"
+
+        buildFile << """
+apply plugin: 'application'
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
+repositories {
+    maven {
+        url 'https://repo.gradle.org/gradle/libs-releases-local'
+    }
+    maven {
+        url '${buildContext.libsRepo.toURI()}'
+    }
+    mavenCentral()
```

Comment:
:x: Should use `mavenCentralRepository()` from parent class.

Reply:
Done.